packages/skills/egg-core/SKILL.md

```
@@ -0,0 +1,192 @@
+  "name": "foo",
+  "eggModule": {
+    "name": "foo"
+  }
+}
+```
+
+**重要提示**：模块名称不能包含 `-` 或其他特殊字符；使用驼峰命名规则。
+
+#### 正确示例
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)